app/server/lib/configureMinIOExternalStorage.ts

```diff
@@ -19,6 +19,11 @@ export function checkMinIOExternalStorage() {
     preferredEnvVar: 'GRIST_DOCS_MINIO_BUCKET',
   });
   if (!bucket) { return undefined; }
+  const region = settings.flag('bucket_region').requireString({
```

Comment:
Grepping for `settings.flag` it looks like it has been likeThisSoFar, so can you switch this to `bucketRegion`?

Reply:
Done ! Excuse my python ;)